lib/recaptcha/http.ex

```diff
@@ -39,10 +39,11 @@ defmodule Recaptcha.Http do
   def request_verification(body, options \\ []) do
     timeout = options[:timeout] || Config.get_env(:recaptcha, :timeout, 5000)
     url = Config.get_env(:recaptcha, :verify_url, @default_verify_url)
+    json = Application.get_env(:recaptcha, :json_library, Poison)
```

Comment:
Is the default value here required? Or we can just have it in the config?

Reply:
Whatever you prefer.

Reply:
I removed it, you were right.